docs/go.mod

```diff
@@ -1,3 +1,9 @@
 module github.com/keptn/keptn-lifecycle-toolkit/docs
 
 go 1.20
+
```

Comment:
why is this introduced ?

Reply:
It was not my choice.  When I did git pull upstream, i got a merge conflict on this file so I fixed the conflict and the two go files became part of my branch.  I don't think Simon is around so I let it go...

Reply:
I removed it recently because it is not needed. Please remove the changes from go.mod and go.sum